### PR TITLE
feature/TSS-1501-barrier-downloads-python-frontend

### PR DESCRIPTION
**Pull request by osimuka**

### CODE REVIEW

**Review by abarolo — APPROVED** [review]

**Comment by abarolo** [comment]
The column width in the downloads table page are mutable at the row level depending on the text length

File: barriers/views/search.py [hunk 1]
```diff
@@ -167,22 +167,83 @@ def get_pageless_querystring(self):
 class DownloadBarriers(SearchFormMixin, View):
     form_class = BarrierSearchForm
 
+    _search_form = None
+
     def get(self, request, *args, **kwargs):
         form = self.form_class(**self.get_form_kwargs())
         form.is_valid()
         search_parameters = form.get_api_search_parameters()
+        search_parameters["filters"] = self.search_form.get_raw_filters()
```
Comment (osimuka) [inline barriers/views/search.py:176]: @abarolo this is were the change is method

Comment (osimuka) [inline barriers/views/search.py:176]: @abarolo this is were the change is method

**Comment by abarolo** [comment]
lgtm seems to be working fine

**Review by Santino-Trade — APPROVED** [review]

**Pull request merged**